src/extensions/core/uploadImage.ts

```diff
@@ -1,13 +1,12 @@
-// @ts-strict-ignore
 import { app } from '../../scripts/app'
-import { ComfyNodeDef } from '@/types/apiTypes'
+import { ComfyObjectInfo } from '@/types/comfy'
```

Comment:
`ComfyObjectInfo` is the legacy type for node definition, we should fix the `ComfyNodeDef` declaration.